exec/jdbc-all/pom.xml

```diff
@@ -1230,7 +1230,7 @@
     <profile>
       <id>hadoop-2</id>
       <properties>
-        <jdbc-all-jar.maxsize>50400000</jdbc-all-jar.maxsize>
+        <jdbc-all-jar.maxsize>52800000</jdbc-all-jar.maxsize>
```

Comment:
Can we exclude some unnecessary dependencies from JDBC Driver to avoid its inflation?

Reply:
Always :-)